irc/src/connection.rs

```diff
@@ -1,8 +1,10 @@
 use std::net::IpAddr;
 use std::path::PathBuf;
+use std::pin::Pin;
 
+use arti_client::DataStream;
```

Comment:
Can we alias this to `TorStream`? `DataStream` is a bit generic to know where it's coming from